test/RuntimeTemplateCheck.test.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
```

Comment:
This is too much code for this test case.

Please use the existing test cases system by putting your test case into `test/cases/...`

Reply:
update, use existing test cases system